### DISCUSSION TITLE
open multiples cases in distinct external HDD's

### DISCUSSION THREAD

**Discussion by thmme** [sepinf-inc/IPED#2314] [General]
Dear all, 

I would like to ask your support with the command line to open multiples cases in distinct external HDD's. 

scenario: HDD1 indexed 10 cases; HDD2 indexed 5 cases. I would like to open all cases in a unique case review. 

Regards,

TM

**Comment by lfcnassif** [comment]
See the wiki manual section about multicases. The command line is there. If you use the txt file parameter, you can specify absolute case paths pointing to different HDDs.
